code/modules/core_implant/cruciform/rituals/construction.dm

```diff
@@ -74,6 +85,52 @@ GLOBAL_LIST_INIT(nt_blueprints, init_nt_blueprints())
 	var/build_path = blueprint.build_path
 	new build_path(target_turf)
 
+/datum/ritual/cruciform/priest/acolyte/deconstruction
+	name = "Uproot"
+	phrase = "Dominus dedit, Dominus abstulit."
+	desc = "Mistakes are to be a lesson, but first we must correct it by deconstructing its form."
+	power = 40
```

Comment:
Why is the power cost of this so high? I hate being cruciform energy cucked...

Reply:
Same as Manifestation, the construction litany.

Litany powers will be looked at later as a whole.